scrapers/hi/bills.py

```diff
@@ -315,6 +319,7 @@ def scrape_bill(self, session, chamber, bill_type, url):
             prior = bill_page.xpath(
                 "//table[@id='ContentPlaceHolderCol1_GridViewStatus']/tr/td/font/text()"
             )[-1]
+            prior_session = "{} Regular Session".format(str(int(session[:4]) - 1))
```

Comment:
Just a nit, but we have f-strings and format-strings peppered through this scraper. Should we settle on one?

Reply:
oops, that's a cut & paste from the previous code. -- I always do f-strings now, will try to watch for them in cut and pastes as well